src/otx/core/data/dataset/visual_prompting.py

```diff
@@ -231,22 +246,24 @@ def _get_item_impl(self, index: int) -> ZeroShotVisualPromptingDataEntity | None
                     )
                     gt_prompts.append(point)
 
-                gt_labels.append(annotation.label)
+                gt_labels["prompts"].append(annotation.label)
+                gt_labels["polygons"].append(annotation.label)
+                gt_labels["masks"].append(annotation.label)
                 gt_masks.append(mask)
                 gt_polygons.append(annotation)
 
             # TODO(sungchul): for mask, bounding box, and point annotation
             elif isinstance(annotation, (dmBbox, dmMask, dmPoints)):
                 pass
 
-        assert len(gt_prompts) > 0, "#prompts must be greater than 0."  # noqa: S101
+        if len(gt_prompts) == 0:
```

Comment:
```suggestion
        if not gt_prompts:
```
It's a minor but how about aligning it to PEP8? This is a reference https://peps.python.org/pep-0008/#programming-recommendations

Reply:
Done 3f1333f